packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -46,6 +47,10 @@ const Checkout = () => {
     const {data: basket} = useCurrentBasket()
     const [isLoading, setIsLoading] = useState(false)
     const {mutateAsync: createOrder} = useShopperOrdersMutation('createOrder')
+    const {passwordless, social} = getConfig().app.login
+    const idps = social?.idps
+    const isSocialEnabled = social?.enabled
+    const isPasswordlessEnabled = passwordless?.enabled
```

Comment:
Same comment-ish from above. These are falsey, so you might want to make them boolean by using !!